contacts.html

```diff
@@ -0,0 +1,468 @@
+            </strong>
+            <p class="introduction__text text__section">
+              We are here to help you navigate your creative journey. Reach out
+              for any questions, support, or feedback.
+            </p>
+          </div>
+          <a
+            href="./index.html"
+            class="about-us__to-shop button button__primary"
+            >Shop Now</a
+          >
+          <div class="about-us__contacts list list__column">
+            <a
+              href="tel:15551234567"
+              target="_blank"
+              class="contacts__contact text__section list"
+            >
+              <svg class="contact__image icon">
+                <use xlink:href="./images/icons/sprite.svg#phone" />
+              </svg>
+              <p class="contact__informtion">+1 (555) 123-4567</p>
+            </a>
+            <a
+              href="https://mail.google.com/mail/?view=cm&fs=1&to=support@pixelmart.com"
+              target="_blank"
+              class="contacts__contact text__section list"
+            >
+              <svg class="contact__image icon">
+                <use xlink:href="./images/icons/sprite.svg#mail" />
+              </svg>
+              <p class="contact__informtion">support@pixmart.com</p>
+            </a>
+            <a
+              href="https://www.google.ru/maps/place/Pixmart/@12.6661441,15.2575709,3z/data=!4m6!3m5!1s0x13258b12119ada97:0x5c2b4c2814205c7!8m2!3d41.7951054!4d12.444515!16s%2Fg%2F11mvmwhcm4?entry=ttu&g_ep=EgoyMDI0MTIxMS4wIKXMDSoASAFQAw%3D%3D"
+              target="_blank"
+              class="contacts__contact text__section list"
+            >
+              <svg class="contact__image icon">
+                <use xlink:href="./images/icons/sprite.svg#clock" />
+              </svg>
+              <p class="contact__informtion">
+                Monday to Friday : 9 AM - 5 PM EST
+              </p>
+            </a>
+          </div>
+          <div class="about-us__map">
+            <a
+              href="https://www.google.ru/maps/place/Pixmart/@12.6661441,15.2575709,3z/data=!4m6!3m5!1s0x13258b12119ada97:0x5c2b4c2814205c7!8m2!3d41.7951054!4d12.444515!16s%2Fg%2F11mvmwhcm4?entry=ttu&g_ep=EgoyMDI0MTIxMS4wIKXMDSoASAFQAw%3D%3D"
+              target="_blank"
+              class="map__address-info"
+            >
+              <svg class="address-info__image icon">
+                <use xlink:href="./images/icons/sprite.svg#pin" />
+              </svg>
+              <strong class="address__name-shop">PixMart HQ</strong>
+              <p class="address__address">
+                123 Creative Ave, Design City, 45678
+              </p>
+            </a>
+          </div>
+        </div>
+        <form action="#" method="post" class="contacts__contact-form">
+          <ul class="contact-form__list list list__column">
+            <li class="contact-form__field list list__column">
+              <label for="personal-info__name">Full name</label>
```

Comment:
We don't use BEM for label/input attributes.
Replace 'personal-info__name' with 'personal-info-name' please

Reply:
Fixed 🥇